lib/validation/validator.js

```diff
@@ -1,26 +1,21 @@
-const path = require("path");
-const {promisify} = require("util");
-const readFile = promisify(require("fs").readFile);
+import {fileURLToPath} from "node:url";
+import {readFile} from "node:fs/promises";
 
 async function loadSchema(schemaPath) {
 	const filePath = schemaPath.replace("http://ui5.sap/schema/", "");
-	const schemaFile = await readFile(path.join(__dirname, "schema", filePath), {encoding: "utf8"});
+	const schemaFile = await readFile(
+		fileURLToPath(new URL(`./schema/${filePath}`, import.meta.url)), {encoding: "utf8"}
+	);
 	return JSON.parse(schemaFile);
 }
 
-/**
- * @private
```

Comment:
`@private` could stay here to highlight thats private

Reply:
Decided to not add it again because there is no further jdoc written and necessary